### DISCUSSION TITLE
i cant make it work ;-; (issue with unban)

### DISCUSSION THREAD

**Discussion by Zaz-Exploit** [Rapptz/discord.py#5958] [Q&A]
so this is the code
in the part await unban(user, *, reason=None)

![imagen](https://user-images.githubusercontent.com/55614820/96773381-6d12ff00-13b2-11eb-9238-3062cb5a10c4.png)
![imagen](https://user-images.githubusercontent.com/55614820/96773403-7603d080-13b2-11eb-9a7c-a663fb340c9b.png)

```@client.command()
    async def unban(self, ctx, *, member=None):
        banned_users = await ctx.guild.bans()
        
        member_name, member_discriminator = member.split('#')
        for ban_entry in banned_users:
            user = ban_entry.user
            
            if (user.name, user.discriminator) == (member_name, member_discriminator):
                await unban(user, *, reason=None)
                embed=discord.Embed(title='', colour=discord.Colour.red())
                embed.add_field(name='Desbaneos', value=f'El usuario {user.mention} fue desbaneado\n**Razon**: {reason}\n\nStaff: **{ctx.author}**')
                await ctx.send(embed=embed)```

**Comment by fwrs** [comment]
1. your submission doesn't appear to showcase a bug with discord.py but rather a problem with your code, in which case github issues aren't the appropriate place to ask for questions regarding your code, use the official [discord server](https://discord.gg/r3sSKJJ)

2. follow the issue template for submitting issues

**Comment by Rapptz** [comment]
The sole asterisk is used only in signatures and means that the arguments following it are keyword only arguments.

```py
def foo(a, *, b):
    pass

# is called like this...
foo(10, b=20)
```
